spec/api-browser-window-spec.ts

```diff
@@ -4671,7 +4671,47 @@ describe('BrowserWindow module', () => {
         expect(w.getChildWindows().length).to.equal(0);
       });
 
-      ifit(process.platform === 'darwin')('child window matches visibility when visibility changes', async () => {
+      ifit(process.platform === 'darwin')('only shows the intended window when a child with siblings is shown', async () => {
+        const w = new BrowserWindow({ show: false });
+        const childOne = new BrowserWindow({ show: false, parent: w });
+        const childTwo = new BrowserWindow({ show: false, parent: w });
+
+        const parentShown = once(w, 'show');
+        w.show();
+        await parentShown;
+
+        expect(childOne.isVisible()).to.be.false('childOne is visible');
+        expect(childTwo.isVisible()).to.be.false('childTwo is visible');
```

Comment:
```suggestion
        expect(childOne.isVisible()).to.be.false('childOne is not visible');
        expect(childTwo.isVisible()).to.be.false('childTwo is not visible');
```

Reply:
this is the intended format - the string should reflect the incorrect condition

e.g. here: 

> expect(childTwo.isVisible()).to.be.false('childTwo is not visible');

what it's saying is: expect the child window not to be visible, if it is, error with a string saying that it _is_ visible